File: KingdomCodeTool/wwwroot/Download/202503250844/Angular/shared/CLSYeuCau_QMS.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CLSYeuCau_QMS } from './CLSYeuCau_QMS.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CLSYeuCau_QMSService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'CLSYeuCau_Id', 'SoTT', 'HangDoi_Idx', 'HangDoi_Details', 'KhuVuc', 'App'];

    List: CLSYeuCau_QMS[] | undefined;
    ListFilter: CLSYeuCau_QMS[] | undefined;
    FormData!: CLSYeuCau_QMS;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CLSYeuCau_QMS";
    }
}

File: KingdomCodeTool/wwwroot/Download/202503250844/Service/Implement/CLSYeuCau_QMSService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class CLSYeuCau_QMSService : BaseService<CLSYeuCau_QMS, ICLSYeuCau_QMSRepository>
    , ICLSYeuCau_QMSService
    {
    private readonly ICLSYeuCau_QMSRepository _CLSYeuCau_QMSRepository;
    public CLSYeuCau_QMSService(ICLSYeuCau_QMSRepository CLSYeuCau_QMSRepository) : base(CLSYeuCau_QMSRepository)
    {
    _CLSYeuCau_QMSRepository = CLSYeuCau_QMSRepository;
    }
    }
    }

